static/js/components/LearnerSearch.js

```diff
@@ -126,6 +129,14 @@ export default class LearnerSearch extends SearchkitComponent {
   countryNameTranslations: Object = makeCountryNameTranslations();
   degreeTranslations: Object = makeDegreeTranslations();
 
+  constructor(props: Object) {
+    super(props);
+    this.WrappedLearnerResult = wrapWithProps(
+      {openLearnerEmailComposer: this.props.openLearnerEmailComposer},
```

Comment:
I don't think it's a good idea to set this in the constructor. What if `openLearnerEmailComposer` changes later?